src/util/schnorr.rs

```diff
@@ -20,6 +20,7 @@
 use core::fmt;
 use prelude::*;
 
+#[deprecated(since = "0.28.0", note = "Please use `util::key` instead")]
 pub use secp256k1::{XOnlyPublicKey, KeyPair};
```

Comment:
Need to use `type` as @sanket1729 said above.

Reply:
done